content/en/news/support/advance-notice-opencensus-deprecation/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Advance notice of Istio deprecating OpenCensus
+subtitle: Support Announcement
+description: Advance notice of Istio deprecating OpenCensus.
+publishdate: 2023-08-17
+---
+
+OpenCensus has [sunset](https://opentelemetry.io/blog/2023/sunsetting-opencensus) on July 31st, 2023.
+
+The OpenCensus sunset will impact Istio's usage of OpenCensus in tracing, logging, and metrics. Istio
+plans to use OpenTelemetry exporters with the OpenTelemetry collector to provide matching telemetry capabilities.
```

Comment:
This is extremely confusing to me.... maybe I am missing something

First, 99.99% of Istio users do not use OpenCensus at all. This document needs to be more clear about who is impacted.

AFAIK only users who configure https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-ExtensionProvider-OpenCensusAgentTracingProvider are impacted. This is also only tracing, not logging/metrics.


its also not clear why otel collector is recommended. That is not our problem - users should use https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-ExtensionProvider-OpenTelemetryTracingProvider, but whether they point to a collector or other OTLP receiver is their problem

Reply:
Stackdriver uses OpenCensus too. We should say clearly which API fields are going to be removed and put deprecations there.

Reply:
This may need to be designed. If an equivalent C++ SDK or an OTLP endpoint is used, the deprecation may be transparent to users.